### PR TITLE
Add the output format in the readme.md

### PR DESCRIPTION
**Pull request by SamDepardieu**
📝 in README.md

### CODE REVIEW

**Comment by nsorros** [comment]
I removed Jonny but I think @lizgzil should review that even though she already did one code review. This is just documentation but she needs to be aware of it.

File: README.md [hunk 1]
```diff
@@ -6,4 +6,40 @@ A web scraper tool to get data for evaluating Wellcome impact.
...
+```
+
+Some providers will have additional parameters:
+
+- WHO:
+
+```
+year: the publication year of the document
+types: an array containing the who type associated with the document
+subjects: an array containing the who subjects of the document
```
Comment (nsorros) [inline README.md]: authors?

File: README.md [hunk 2]
```diff
@@ -6,4 +6,40 @@ A web scraper tool to get data for evaluating Wellcome impact.
...
+It will always have the following attribute, thought:
+
+```
+title: a string containing the document title
+uri: the url of the document
+pdf: the name of the file
+sections: a json object of section names, containing the text extracted from matching sections
+keywords: a json object of keywords, containing the text extracted from matching text
+hash: a md5 digest of the file
+provider: the provider from where the file has been downloaded
```
Comment (nsorros) [inline README.md]: you can also use tables if you think they are better
```
| Attribute        | Description                                                |
| ------------- | --------------------------------------- |
| title                | a string containing the document title    |
| uri                  | the url of the document                            |
```

**Comment by SamDepardieu** [comment]
That's correct, I'll add authors. Tables looks great, I'll use them 👍

File: README.md [hunk 3]
```diff
@@ -6,4 +6,39 @@ A web scraper tool to get data for evaluating Wellcome impact.
...
 ## Usage
 
-See the wiki: https://github.com/wellcometrust/wsf-web-scraper/wiki
+To deploy this scraper yourself, see the wiki: https://github.com/wellcometrust/wsf-web-scraper/wiki
+
+This scraper can also be deployed more easily using Docker.
+
+## Output Formating
+
+The outputed file is meant to contains a number a different fields, which can vary depending on the scraped provider.
```
Comment (lizgzil) [inline README.md]: 'contain'

File: README.md [hunk 4]
```diff
@@ -6,4 +6,39 @@ A web scraper tool to get data for evaluating Wellcome impact.
...
+
+It will always have the following attribute, thought:
```
Comment (lizgzil) [inline README.md]: 'attributes, though'

File: README.md [hunk 5]
```diff
@@ -6,4 +6,39 @@ A web scraper tool to get data for evaluating Wellcome impact.
...
+|year     | the publication year of the document|
+|types    | an array containing the who type associated with the document|
+|subjects | an array containing the who subjects of the document|
+|authors  | an array containing the authors (from WHO)|
+
+### Nice
+
+|Attribute|Description|
+|---------|-----------|
+|year     | the publication year of the document|
```
Comment (lizgzil) [inline README.md]: do you anticipate that they might not all have years then?

File: README.md (hunk 3 above)
Comment (lizgzil) [reply README.md]: 'scraper'

File: README.md [hunk 6]
```diff
@@ -6,4 +6,39 @@ A web scraper tool to get data for evaluating Wellcome impact.
...
+|provider | the provider from where the file has been downloaded|
+
+Some providers will have additional parameters:
+
+### WHO
+
+|Attribute|Description|
+|---------|-----------|
+|year     | the publication year of the document|
+|types    | an array containing the who type associated with the document|
```
Comment (lizgzil) [inline README.md]: capital WHO

File: README.md [hunk 7]
```diff
@@ -6,4 +6,39 @@ A web scraper tool to get data for evaluating Wellcome impact.
...
+|subjects | an array containing the who subjects of the document|
```
Comment (lizgzil) [inline README.md]: capital WHO

File: README.md (hunk 5 above)
Comment (SamDepardieu) [reply README.md]: It's a sure thing, I didn't manage to get the years for uncief, gov and msf

Comment (lizgzil) [reply README.md]: ah that's annoying! Good to know.

Comment (SamDepardieu) [reply README.md]: If it's very important for you to have it, I probably manage somehow, but it's not easy on the three new websites

File: README.md [hunk 8]
```diff
@@ -6,4 +6,39 @@ A web scraper tool to get data for evaluating Wellcome impact.
...
+It will always have the following attributes, though:
+
+|Attribute|Description|
+|---------|-----------|
+|title    | a string containing the document title|
+|uri      | the url of the document|
+|pdf      | the name of the file|
+|sections | a json object of section names, containing the text extracted from matching sections|
+|keywords | a json object of keywords, containing the text extracted from matching text|
+|hash     | a md5 digest of the file|
```
Comment (nsorros) [inline README.md]: Is it possible to specificy which field can be used as a unique key?

**Comment by SamDepardieu** [comment]
@lizgzil I'll take some time this week to see if I can make `year` a generic field, as it seems to be important for you 👍

**Pull request merged**